app/assets/stylesheets/components/_alert.scss

```diff
@@ -8,10 +8,10 @@
   color: white;
 }
 .alert-info {
-  background: $green;
```

Comment:
Something wrong with Sass color variables?

Reply:
I removed all Sass color variables not in our chart

Reply:
OK